eng/Subsets.props

```diff
@@ -150,6 +150,7 @@
                              $(CoreClrProjectRoot)src\tools\dotnet-pgo\dotnet-pgo.csproj;
                              $(CoreClrProjectRoot)src\tools\r2rtest\R2RTest.csproj" Category="clr" BuildInParallel="true" />
     <ProjectToBuild Include="$(CoreClrProjectRoot)src\tools\aot\crossgen2\crossgen2.csproj" Category="clr" />
+    <ProjectToBuild Include="$(CoreClrProjectRoot)src\tools\aot\ILCompiler.TypeSystem.ReadyToRun.Tests\ILCompiler.TypeSystem.ReadyToRun.Tests.csproj" Test="true" Category="clr" />
```

Comment:
I don't believe we need the `Test="true"` attribute here unless the intent is to only build this project when we're running the tests.

Reply:
It looks required; I tried removing it and the tests stopped running with `build.cmd -test`. With `Test="true"`, I still see the tests getting built without `-test` passed to build.